### DISCUSSION TITLE
Macbook Tick Health plugin

### DISCUSSION THREAD

**Discussion by HelveticaStandardBold** [runelite/runelite#17108] [Ideas]
Macbooks have very aggressive wireless adapter power saving settings that are unable to be altered. They are so aggressive that it can turn on and off in between the runescape 0.6s tick interval. Without anything else running in the background that uses the network, i usually get around 60-75% tick health. There is a workaround in which you run 'ping 192.168.1.1 -i 0.2' in the terminal so you are pinging yourself every 0.2 seconds and with this i am instantly elevated to 97%+ tick health. While this is easy enough it seems that there could be a more simple solution.

My suggestion is a plugin that does the exact same thing but can be instead enabled with the click of a button. I assume all it would take is the runelite client sending a ping to your IP every 0.2 seconds to achieve this. Does this seem feasible?

**Comment by Felanbird** [comment]
Already PR'd in [runelite/pluginhub#4830](https://github.com/runelite/plugin-hub/pull/4830) - the hub queue is just very behind, especially for new plugins.

**Reply by HelveticaStandardBold** [reply]
My bad, i searched but never came across that thread. Thanks for letting me know.
